docs/exterminate.rst

```diff
@@ -37,14 +33,30 @@ Examples
     Kill the selected unit.
 ``exterminate``
     List the targets on your map.
-``exterminate BIRD_RAVEN:male``
+``exterminate BIRD_RAVEN:MALE``
     Kill the ravens flying around the map (but only the male ones).
-``exterminate undead magma``
-    Kill all undead on the map by pouring magma on them.
+``exterminate GOBLIN --method MAGMA --only-visible --only-hostile``
+    Kill all visible, hostile goblins on the map by drowning them in magma.
+
+Options
+-------
+
+``--method <method>``
+    Specifies the "method" of killing units. See ``exterminate --help`` for a
+    list of possible methods.
+``--only-visible``
+    Specifies the tool should only kill units that are visible to the player
+    on the map.
+``--only-hostile``
+    Specifies the tool should only kill units that are hostile to the player.
```

Comment:
it might be useful to make this behavior the default, and instead have a ``--include-friendly`` option. Players *probably* don't intend to kill the visiting goblin bards when they wipe out a siege.